nwb_conversion_tools/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py

```diff
@@ -78,9 +107,10 @@ def get_metadata(self):
                     )
                 )
 
+        # CellClass file
         celltype_filepath = session_path / f"{session_id}.CellClass.cellinfo.mat"
```

Comment:
```suggestion
        celltype_filepath = session_path / f"{session_id}.CellClass.cellinfo.mat"
```
Clear enough without added line/comment.

I guess we don't have any GIN data for the CellClass functionality do we? Can you add that as a low priority item to your to-do list?